platforms/blabsy/src/components/tweet/tweet-share.tsx

```
@@ -13,115 +13,127 @@ import { ToolTip } from '@components/ui/tooltip';
-              </span>
-            )
-          : 'Tweet removed from your bookmarks'
-      );
-    };
+            toast.success(
+                type === 'bookmark'
+                    ? (): JSX.Element => (
+                          <span className='flex gap-2'>
+                              Tweet added to your Bookmarks
+                              <Link href='/bookmarks'>
+                                  <a className='custom-underline font-bold'>
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)